src/typings/vscode.proposed.d.ts

```diff
@@ -16,6 +16,60 @@
 
```

Comment:
can you copy this file in one more time? looks like `env.uriScheme` has been promoted to stable so there are errors about it being redeclared here